assessment_helpers/to_csv.py

```diff
@@ -0,0 +1,54 @@
+#This script formats the "GAdata.txt" file (created by trainGeneticAlgorithm.py) as a .csv file named "ga_data.csv"
+#Note: make sure "GAdata.txt" is named exactly that and is in the same directory as this script
```

Comment:
This comment might be deprecated by now

Reply:
probably would be helpful to include information about how to use the file in the comment